code/modules/mod/mod_types.dm

```diff
@@ -477,6 +477,25 @@
 		/obj/item/mod/module/quick_cuff,
 	)
 
+/obj/item/mod/control/pre_equipped/responsory/inquisitory/syndie
+	starting_frequency = MODLINK_FREQ_SYNDICATE
+	applied_cell = /obj/item/stock_parts/cell/super
+	applied_modules = list(
+		/obj/item/mod/module/storage/syndicate,
+		/obj/item/mod/module/emp_shield,
+		/obj/item/mod/module/magnetic_harness,
+		/obj/item/mod/module/jetpack,
+		/obj/item/mod/module/pathfinder,
+		/obj/item/mod/module/flashlight/darkness,
+		/obj/item/mod/module/visor/night,
+		/obj/item/mod/module/dna_lock,
+		/obj/item/mod/module/quick_cuff,
+		/obj/item/mod/module/visor/night,
+		/obj/item/mod/module/shove_blocker,
+		/obj/item/mod/module/noslip,
+		/obj/item/mod/module/anti_magic,
```

Comment:
I know the ERT suit has this but that's an admin item, it feels very weird for traitor gear to have this.

Reply:
Ok, I'll remove it.